tooling/build-and-use.mjs

```diff
@@ -1,69 +1,129 @@
 import { runTask } from './taskrunner.mjs';
 import chalk from 'chalk';
-import { readdir } from "fs/promises";
-import * as readline from 'node:readline/promises'
-import { stdin as input, stdout as output, } from 'node:process'
+import { readdir } from 'fs/promises';
+import * as readline from 'node:readline/promises';
+import { stdin as input, stdout as output } from 'node:process';
 
 async function runTasks() {
-    console.clear()
-    console.log(chalk.bold.yellow('Running build and use tasks...\n'))
+  console.clear();
+  const presetApp = process.argv.at(2).match(/--(\w+-?)+/)?.[1];
```

Comment:
when not supplying an argument, the script returns:
```
const presetApp = process.argv.at(2).match(/--(\w+-?)+/)?.[1];
                                      ^
TypeError: Cannot read properties of undefined (reading 'match')
```
and then exits, making it not usable unless you supply an argument. 
I guess its fine that you always have to supply an argument, but in that case the error should be caught and supplied with instructions on what to do. Another solution would be just to display the list.